File: Copilot/Java/Factory_Method/round3/source/Beverages.java
public class Espresso implements MenuItem {
    @Override
    public void serve() {
        System.out.println("Serving Espresso");
    }
}

public class Latte implements MenuItem {
    @Override
    public void serve() {
        System.out.println("Serving Latte");
    }
}

public class Milkshakes implements MenuItem {
    @Override
    public void serve() {
        System.out.println("Serving Milkshakes");
    }
}

public class FruitJuices implements MenuItem {
    @Override
    public void serve() {
        System.out.println("Serving Fruit Juices");
    }
}

File: Copilot/Python/Factory_Method/round2/source/beverages.py
from menu_item import MenuItem

class Espresso(MenuItem):
    def serve(self):
        print("Serving Espresso")

class Latte(MenuItem):
    def serve(self):
        print("Serving Latte")

class Milkshakes(MenuItem):
    def serve(self):
        print("Serving Milkshakes")

class FruitJuices(MenuItem):
    def serve(self):
        print("Serving Fruit Juices")
